crm_project/wizard/crm_lead_convert2task.py

```diff
@@ -41,6 +42,12 @@ def action_lead_to_project_task(self):
         task = self.env["project.task"].create(vals)
         # move the mail thread
         lead.message_change_thread(task)
+        # add followers from lead
+        for follower in lead.message_follower_ids:
+            task.message_subscribe(
+                partner_ids=[follower.partner_id.id],
+                subtype_ids=follower.subtype_ids.ids,
```

Comment:
Only shared subtypes should be transferred. Not sure if this is done by upper code.

Reply:
Other approach can be to subscribe with default types.